source/index.html.md.erb

```diff
@@ -54,6 +54,7 @@ are still working for GDS as an organization.
 - [sending emails](standards/sending-email.html)
 - [storing and querying logs](standards/logging.html)
 - [supporting a service](standards/supporting-services.html)
+- [testing a service](standards/testing-services.html)
```

Comment:
testing a service is quite ambiguous, it could mean acceptance testing or anything really.

Maybe "Penetration testing a service" or "Security testing a service"

Reply:
Done.